proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,393 @@
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Retrieve the mount task status"
+      description:
+        "This operation retrieves the status of a task responsible for mounting or unmounting topics. "
+        "It provides details on the task’s type (mount or unmount), its current state, and the topics involved."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.ListMountTasksResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc GetMountTask(GetMountTaskRequest) returns (GetMountTaskResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha2/cloud-storage/mount-tasks/{id}"
+      response_body: "task"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Retrieve the mount task status"
+      description:
+        "This operation retrieves the status of a task responsible for mounting or unmounting topics. "
+        "It provides details on the task’s type (mount or unmount), its current state, and the topics involved."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.GetMountTaskResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc DeleteMountTask(DeleteMountTaskRequest) returns (DeleteMountTaskResponse) {
+    option (google.api.http) = {
+      delete: "/v1alpha2/cloud-storage/mount-tasks/{id}"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Delete a mount task"
+      description: "This operation deletes a mount task."
+      responses: {
+        key: "202"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.DeleteMountTaskResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc UpdateMountTask(UpdateMountTaskRequest) returns (UpdateMountTaskResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/mount-tasks/{id}"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Update a mount task"
```

Comment:
Is this one also for both mount and unmount?

Reply:
Yes